packages/python/test/fixtures/01-cowsay/vercel.json

```diff
@@ -1,8 +1,9 @@
 {
   "version": 2,
-  "builds": [{ "src": "**/**.py", "use": "@vercel/python" }],
-  "probes": [
-    { "path": "/", "mustContain": "pip:RANDOMNESS_PLACEHOLDER" },
-    { "path": "/another", "mustContain": "upper:RANDOMNESS_PLACEHOLDER" }
+  "routes": [
+    {
+      "src": "/(.*)",
```

Comment:
Huh, yeah. Turns out that some of these fixtures were serving the files as raw files, but the check happened to pass because `RANDOMNESS_PLACEHOLDER` was replaced in them.

I updated those fixtures to make sure the endpoints are either in `api` or specify a builder to use. I added content type checks to the probes to make sure none were rendered as raw files.

Reply:
Its probably best to update the `probes.json` to test the actual and then you can remove `vercel.json` completely

Reply:
Yeah, that is better. I removed the `vercel.json` files I could and updated paths appropriately: [`ddff264` (#9668)](https://github.com/vercel/vercel/pull/9668/commits/ddff26450ba6354ed9519151c2d316cb0633db6c)